### PR TITLE
Fire issue events on HA's bus

### PR DESCRIPTION
**Pull request by mdegat01**
## Proposed change

Follow up to #3818 - the events need to be on HA's bus, not supervisor's. HA isn't listening to supervisor's event bus, supervisor fires events of type `supervisor/event` when HA needs to know about it.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:
- Link to cli pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/homeassistant/websocket.py [hunk 1]
```diff
@@ -279,3 +279,28 @@ def send_message(self, message: dict[str, Any]) -> None:
...
+                    ATTR_DATA: {
+                        ATTR_EVENT: event,
+                        ATTR_DATA: data or {},
+                    },
+                }
+            )
+        except HomeAssistantWSNotSupported:
+            pass
+        except HomeAssistantWSError as err:
+            _LOGGER.error(err)
```
Comment (pvizeli) [inline supervisor/homeassistant/websocket.py:300]: Let's add a message for humans and not only an err

**Review by pvizeli — APPROVED** [review]

File: supervisor/homeassistant/websocket.py (hunk 1 above)
Comment (mdegat01) [reply supervisor/homeassistant/websocket.py]: @pvizeli Changed both places (`async_supervisor_event` and `async_supervisor_update_event`) to this:
```py
_LOGGER.error("Could not send message to Home Assistant due to %s", err)
```
Sound good?

Comment (ludeeus) [reply supervisor/homeassistant/websocket.py]: There is no need for exception handling in both those, one of them should call the other

Comment (mdegat01) [reply supervisor/homeassistant/websocket.py]: Right that's true. I'll adjust it in a follow-up

**Pull request merged**